tslib not set as a dependency

**Type of issue:**  (check with "[x]")
```
- [ ] New feature request
- [X] Bug  
- [ ] Support request
```
**Current behavior:**
After I npm install and trying one of the examples the `tslib` package is not installed and I get an error (`Error: Cannot find module 'tslib'`). I have to install it manually.

**Expected behavior:**
I would expect all the dependencies necessary for this package to work to be set as dependencies in the package.json so I don't have to install anything manually.
 I
**Steps to reproduce the issue:**
- npm install @alfresco/js-api
- script:
```
 const { AlfrescoApi, SearchApi, NodesApi, ContentApi } = require("@alfresco/js-api");
 const alfrescoApi = new AlfrescoApi();
```
**Node version (for build issues):**  
Node version: v16.13.2

--- a/package.json
+++ b/package.json
@@ -34,7 +34,8 @@
   "dependencies": {
     "event-emitter": "^0.3.5",
     "minimatch": "5.0.1",
-    "superagent": "^6.0.0"
+    "superagent": "^6.0.0",
+    "tslib": "^2.0.0"
   },
   "devDependencies": {
     "@types/chai": "^4.2.3",

